packages/hub/src/users.spec.ts

```diff
@@ -18,36 +18,43 @@ import { Client } from './users'
 const addrApiurl = 'http://127.0.0.1:3007'
 const addrGatewayUrl = 'http://127.0.0.1:8006'
 const wrongError = new Error('wrong error!')
-const sessionSecret = 'textilesession'
+const sessionSecret = 'hubsession'
 
 describe('Users...', () => {
   describe('getThread', () => {
     const ctx = new Context(addrApiurl, undefined)
     const client = new Client(ctx)
     let dev: SignupReply.AsObject
     before(async function () {
-      this.timeout(3000)
+      this.timeout(10000)
       const { user } = await signUp(ctx, addrGatewayUrl, sessionSecret)
       if (user) dev = user
     })
-    it('should handle bad keys', async () => {
-      // No key
+    it('should handle bad user group keys', async () => {
+      /**
+       * No key will fail with unauthorized since a key is the minimum
+       * authorization
+       */
       try {
         await client.getThread('foo', ctx)
         throw wrongError
       } catch (err) {
         expect(err).to.not.equal(wrongError)
         expect(err.code).to.equal(grpc.Code.Unauthenticated)
       }
-      // No key signature
+      /**
+       * No key signature
+       * This will fail with NotFound due to it needing to know the key's
+       * security status before it knows if it's authorized or not.
+       */
       const tmp = new Context(addrApiurl).withSession(dev.session)
       const key = await createKey(tmp, 'ACCOUNT')
       try {
         await client.getThread('foo', ctx.withAPIKey(key.key))
         throw wrongError
       } catch (err) {
         expect(err).to.not.equal(wrongError)
-        expect(err.code).to.equal(grpc.Code.Unauthenticated)
+        expect(err.code).to.equal(grpc.Code.NotFound)
```

Comment:
are we hoping that will be reverted to NotFound in a subsequent textile release?

Reply:
This is the current case. We want to be reverted to Unauthenticated